common/changes/office-ui-fabric-react/SpinButton-ExposeKeyPressEventToOnValidate_2018-08-24-02-22.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Expose keyPress event used to commit manually entered value to OnValidate callback so consumers can handle focus in an accessible way",
```

Comment:
I think the comment needs to be slightly updated since we aren't only exposing the keypress event now